HARK/parallel.py

```diff
@@ -335,30 +351,34 @@ def saveNelderMeadData(name, simplex, fvals, iters, evals):
         The number of completed Nelder-Mead iterations.
     evals : int
         The cumulative number of function evaluations in the search process.
-
+        
     Returns
     -------
-    none
+    None
     '''
-    f = open(name + '.txt','w')
-    my_writer = csv.writer(f,delimiter=' ')
+    N = simplex.shape[0] # Number of points in simplex
+    K = simplex.shape[1] # Total number of parameters
+    
+    f = open(name + '.txt','wb')
```

Comment:
This could be something like

``` python
with open(name + '.txt','wb') as f:
    my_writer = csv.writer(f,delimiter=',')
    my_writer.writerow(simplex.shape)
    my_writer.writerow([iters, evals])
    for n in range(N):
        my_writer.writerow(simplex[n,:])
    my_writer.writerow(fvals)
```
reduces the probability of missing the open and close of the file in future iterations.